.eslintrc.json

```diff
@@ -22,6 +22,7 @@
         "no-multiple-empty-lines": 1,
         "prefer-const": "error",
         "space-infix-ops": "error",
-        "no-useless-escape": "off"
+        "no-useless-escape": "off",
+        "no-var": "error"
```

Comment:
@dplewis the `no-var`  should go into another PR. The local datastore is a big enough PR like that. I'd rather get the linting done with `no-var` done at a later time. Here it pollutes the PR and the feature. So please put this commit into another branch for now.

Reply:
I got it.